### PR TITLE
tidb-in-kubernetes: make deploy doc clearer on aliyun

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? 
make deploy doc clearer on aliyun

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 1]
```diff
@@ -206,10 +206,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
 通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是，`operator_helm_values` 配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
 
+- 在 `terraform.tfvars` 中设置 `operator_helm_values`：
+
 ```hcl
-variable "operator_helm_values" {
-  default = file("my-operator-values.yaml")
-}
+operator_helm_values = "./my-operator-values.yaml"
+```
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]: Please indent this code block with 4 spaces.

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 2]
```diff
@@ -206,10 +206,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+
+- 在 `main.tf` 中设置 `operator_helm_values`：
+
+```hcl
+operator_helm_values = file("./my-operator-values.yaml")
 ```
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]: Please indent this code block with 4 spaces.

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 3]
```diff
@@ -207,10 +207,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
 通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是，`operator_helm_values` 配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
 
+- 在 `terraform.tfvars` 中设置 `operator_helm_values`：
+
 ```hcl
-variable "operator_helm_values" {
-  default = file("my-operator-values.yaml")
-}
+operator_helm_values = "./my-operator-values.yaml"
+```
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]: Please indent this code block with 4 spaces.

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 4]
```diff
@@ -207,10 +207,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+
+- 在 `main.tf` 中设置 `operator_helm_values`：
+
+```hcl
+operator_helm_values = file("./my-operator-values.yaml")
 ```
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]: ditto

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md (hunk 3 above)
Comment (DanielZhangQD) [reply v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]: done

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md (hunk 1 above)
Comment (DanielZhangQD) [reply dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]: done

**Comment by DanielZhangQD** [comment]
@lilin90 comments addressed, PTAL again.

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md (hunk 4 above)
Comment (DanielZhangQD) [reply v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]: done

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md (hunk 3 above)
Comment (DanielZhangQD) [reply v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]: done

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md (hunk 2 above)
Comment (DanielZhangQD) [reply dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]: done

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 5]
```diff
@@ -206,10 +206,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
 
 通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是，`operator_helm_values` 配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
 
+- 在 `terraform.tfvars` 中设置 `operator_helm_values`：
+
 ```hcl
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```hcl
```

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 6]
```diff
@@ -206,10 +206,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
-variable "operator_helm_values" {
-  default = file("my-operator-values.yaml")
-}
+    operator_helm_values = "./my-operator-values.yaml"
+```
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```
```

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 7]
```diff
@@ -206,10 +206,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+
+- 在 `main.tf` 中设置 `operator_helm_values`：
+
+```hcl
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```hcl
```

File: dev/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 8]
```diff
@@ -206,10 +206,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+    operator_helm_values = file("./my-operator-values.yaml")
 ```
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```
```

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 9]
```diff
@@ -207,10 +207,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
 
 通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是，`operator_helm_values` 配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
 
+- 在 `terraform.tfvars` 中设置 `operator_helm_values`：
+
 ```hcl
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```hcl
```

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 10]
```diff
@@ -207,10 +207,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
-variable "operator_helm_values" {
-  default = file("my-operator-values.yaml")
-}
+    operator_helm_values = "./my-operator-values.yaml"
+```
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```
```

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 11]
```diff
@@ -207,10 +207,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+
+- 在 `main.tf` 中设置 `operator_helm_values`：
+
+```hcl
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```hcl
```

File: v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md [hunk 12]
```diff
@@ -207,10 +207,16 @@ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+    operator_helm_values = file("./my-operator-values.yaml")
 ```
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/deploy/alibaba-cloud.md]:
```suggestion
    ```
```

**Comment by lilin90** [comment]
@aylei PTAL

**Pull request merged**